### ISSUE TITLE
Splice-Vaihingen dataset

### ISSUE THREAD

**Issue by zth1100** [littlebeen/Forgery-localization-for-remote-sensing#2]
Can you provide the dataset Splice-Vaihingen，thanks

**Comment by yejy53** [comment]
+1

**Comment by littlebeen** [comment]
Well, since we don't pay much attention to
![top_mosaic_09cm_area10_tif_1](https://github.com/user-attachments/assets/641354c9-9ae9-41ee-82c4-f1145730a245)
 this dataset, it is a quite coarse like this. Do you still want it?

**Comment by zth1100** [comment]
Yes, I still want it.If you are convenient, please give me a link to download. Thankyou very much

**Comment by littlebeen** [comment]
Its masks are all similar with Fake-Vaihingen.
